paxos/acceptor.cc

```diff
@@ -70,6 +70,7 @@ Status AcceptorImpl::Prepare(ServerContext* context,
     response->set_accepted_value(entry.accepted_value_);
     response->set_has_accepted_value(true);
   }
+  response->set_max_chosen_index(this->replicated_log_->GetLogEntries().rbegin()->first);
```

Comment:
This is not the max chosen idx. This is just the max idx (can be accepted also).
This should be named max_idx_in_log or something

Reply:
fixed